src/Microsoft.SqlTools.ServiceLayer/Localization/sr.Designer.cs

```diff
@@ -258,47 +258,281 @@ public class sr {
+        public static string EditDataQueryNotCompleted {
+            get {
+                return ResourceManager.GetString("EditDataQueryNotCompleted", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///    Looks up a localized string similar to Given row ID is outside the range of rows in the edit cache.
+        /// </summary>
+        public static string EditDataRowOutOfRange {
+            get {
+                return ResourceManager.GetString("EditDataRowOutOfRange", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///    Looks up a localized string similar to An output filename must be provided.
+        /// </summary>
+        public static string EditDataScriptFilePathNull {
+            get {
+                return ResourceManager.GetString("EditDataScriptFilePathNull", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///    Looks up a localized string similar to Edit session already exists..
+        /// </summary>
+        public static string EditDataSessionAlreadyExists {
+            get {
+                return ResourceManager.GetString("EditDataSessionAlreadyExists", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///    Looks up a localized string similar to Edit session has already been initialized.
+        /// </summary>
+        public static string EditDataSessionAlreadyInitialized {
+            get {
+                return ResourceManager.GetString("EditDataSessionAlreadyInitialized", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///    Looks up a localized string similar to Edit session has already been initialized or is in the process of initializing.
+        /// </summary>
+        public static string EditDataSessionAlreadyInitializing {
+            get {
+                return ResourceManager.GetString("EditDataSessionAlreadyInitializing", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///    Looks up a localized string similar to Edit session does not exist..
+        /// </summary>
+        public static string EditDataSessionNotFound {
+            get {
+                return ResourceManager.GetString("EditDataSessionNotFound", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///    Looks up a localized string similar to Edit session has not been initialized.
+        /// </summary>
+        public static string EditDataSessionNotInitialized {
+            get {
+                return ResourceManager.GetString("EditDataSessionNotInitialized", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///    Looks up a localized string similar to TIME column values must be between 00:00:00.0000000 and 23:59:59.9999999.
+        /// </summary>
+        public static string EditDataTimeOver24Hrs {
```

Comment:
I've sync'd the latest code from dev. It seems the checkin run srgen and include the latest sr.Designer.cs

---
In reply to: [111650561](https://github.com/Microsoft/sqltoolsservice/pull/316#discussion_r111650561) [](ancestors = 111650561)